src/openrct2/audio/audio.h

```diff
@@ -55,20 +55,21 @@ struct rct_ride_music_params
     uint16_t frequency;
 };
 
+struct Sound
```

Comment:
I feel as though we might get name collisions on this name. Also not this structure is also used in `rct_ride_music`.

Reply:
We should start looking into namespaces to avoid name collisions :)

I'll work on making rct_ride_music use Sound as well in a future PR

Reply:
Actually, rct_ride_music has a field ```uint8_t tune_id```, instead of a ```SoundId Id``` (the underlying type is the same)

Reply:
Yes namespaces would be good.